src/distances/alignment_helpers.rs

```diff
@@ -0,0 +1,559 @@
+                    (9, Direction::Diagonal)
+                ],
+                [
+                    (14, Direction::None),
+                    (13, Direction::Up),
+                    (12, Direction::Up),
+                    (11, Direction::Up),
+                    (10, Direction::Up),
+                    (9, Direction::Up),
+                    (8, Direction::Up),
+                    (7, Direction::Diagonal),
+                    (6, Direction::Up),
+                    (5, Direction::Up),
+                    (4, Direction::Diagonal),
+                    (5, Direction::Left),
+                    (6, Direction::Left),
+                    (6, Direction::Diagonal),
+                    (7, Direction::Left),
+                    (8, Direction::Left),
+                    (9, Direction::Diagonal)
+                ],
+                [
+                    (15, Direction::None),
+                    (14, Direction::Up),
+                    (13, Direction::Up),
+                    (12, Direction::Up),
+                    (11, Direction::Up),
+                    (10, Direction::Up),
+                    (9, Direction::Up),
+                    (8, Direction::Up),
+                    (7, Direction::Up),
+                    (6, Direction::Up),
+                    (5, Direction::Up),
+                    (4, Direction::Diagonal),
+                    (5, Direction::Left),
+                    (6, Direction::Left),
+                    (7, Direction::Diagonal),
+                    (8, Direction::Diagonal),
+                    (9, Direction::Diagonal)
+                ],
+                [
+                    (16, Direction::None),
+                    (15, Direction::Up),
+                    (14, Direction::Up),
+                    (13, Direction::Up),
+                    (12, Direction::Up),
+                    (11, Direction::Up),
+                    (10, Direction::Up),
+                    (9, Direction::Up),
+                    (8, Direction::Up),
+                    (7, Direction::Up),
+                    (6, Direction::Up),
+                    (5, Direction::Up),
+                    (4, Direction::Diagonal),
+                    (5, Direction::Left),
+                    (6, Direction::Left),
+                    (7, Direction::Left),
+                    (8, Direction::Diagonal)
+                ]
+            ]
+        );
+    }
+
+    #[test]
+    fn test_traceback_recursive() {
+        let x_u8 = "NAJIBPEPPERSEATS".as_bytes();
+        let y_u8 = "NAJIBEATSPEPPERS".as_bytes();
+        let table1: Vec<Vec<(usize, Direction)>> = compute_nw_table(x_u8, y_u8);
+        let nw_u8 = traceback_recursive(&table1, (x_u8, y_u8));
+        assert_eq!(
+            nw_u8,
+            (
+                [78, 65, 74, 73, 66, 45, 80, 69, 80, 80, 69, 82, 83, 69, 65, 84, 83].to_vec(),
+                [78, 65, 74, 73, 66, 69, 65, 84, 83, 80, 69, 80, 80, 69, 45, 82, 83].to_vec()
+            )
+        );
```

Comment:
What do you see if you convert these types back to `str`? It's hard to intuitively recognize the answer when looking at a sequence of integers. Try doing the same for the rest of the tests.